conf/tufts.config

```diff
@@ -33,3 +33,17 @@ singularity {
         enabled = true
         autoMounts = true
 }
+
+// Perform work directory cleanup when the run has succesfully completed
+trace {
+        trace.overwrite = true
+        enabled         = true
+}
+
+// On a successful completion of a Nextflow run, automatically delete all intermediate files stored in the work/ directory
+cleanup = true
```

Comment:
Are you sure you really want this to happen by default?

Reply:
And it only happens if run completes successfully, which in most cases you assume it should be fine to delte them :+1:

Reply:
> Are you sure you really want this to happen by default?

Our cluster does not have a purge policy, and the storage is free. Right now limited storage is the No.1 issue. Thanks for advice.